ioc/tests/support/StreamDevice/simple/top/simpleApp/Db/simple.db

```diff
@@ -1,81 +1,82 @@
-# Fixed values
+# Normal flow
 
-record(ai, "FLOAT:IN") {
-	field(DESC, "A fixed float")
+record(ao, "${P}UCmd") {
+	field(DESC, "Voltage command")
 	field(DTYP, "stream")
-	field(INP, "@simple.proto getFloat $(PORT)")
-	field(PINI, "YES")
-	field(SCAN, "5 second")
+	field(OUT, "@simple.proto setUCmd $(PORT)")
+	field(EGU, "A")
```

Comment:
A is for a measure of intensity not Voltage. Same for the other voltages measure of current